### PR TITLE
Adding Teradici PCoIP Client to Cask repository.

### PR DESCRIPTION
**Pull request by dbhagen**
Signed-off-by: Daniel B. Hagen <daniel.b.hagen@gmail.com>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

---

The only issue I'm having is on the uninstall. It's a `.app` copied out of a `.dmg`, but when you run the app it asks for sudo/admin creds and modifies the application folder structure, such that sudo is required to remove it. I have yet to find an uninstaller inside the package (not mine), so I don't have anything to fire as a script with sudo enabled. What uninstall step would work for this setup?

### CODE REVIEW

File: Casks/teradici-pcoip-client.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask "teradici-pcoip-client" do
+  version "2020.07"
+  sha256 "0a494414547c1bf66a843f21d0a14edad1d1abcf1994dd3159b304556239cd3c"
+
+  url "https://downloads.teradici.com/mac/stable/pcoip-client_20.07.0.dmg"
```
Comment (miccal) [inline Casks/teradici-pcoip-client.rb:5]: Please change this to:
```
url "https://downloads.teradici.com/mac/stable/pcoip-client_#{version}.dmg"
```
and please add the following `appcast`:
```
appcast "https://docs.teradici.com/documentation/software-client-for-macos/release-notes"
        must_contain: version.major_minor
```

File: Casks/teradici-pcoip-client.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+cask "teradici-pcoip-client" do
+  version "2020.07"
```
Comment (miccal) [inline Casks/teradici-pcoip-client.rb:2]: Please change this to:
```
version "20.07.0"
```

File: Casks/teradici-pcoip-client.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  sha256 "0a494414547c1bf66a843f21d0a14edad1d1abcf1994dd3159b304556239cd3c"
```
Comment (miccal) [inline Casks/teradici-pcoip-client.rb:3]: Please change this to:
```
sha256 "70a2cbb8e329cb459ff076ed754e44d1f168d291714bd4f4c5efebb87902959d"
```

File: Casks/teradici-pcoip-client.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+cask "teradici-pcoip-client" do
+  version "20.07.2"
+  sha256 "70a2cbb8e329cb459ff076ed754e44d1f168d291714bd4f4c5efebb87902959d"
+
+  url "https://downloads.teradici.com/mac/stable/pcoip-client_#{version}.dmg"
```
Comment (miccal) [inline Casks/teradici-pcoip-client.rb:5]: Please don't forget to add the following `appcast`:
```
appcast "https://downloads.teradici.com/mac/stable/"
```

Comment (dbhagen) [reply Casks/teradici-pcoip-client.rb:5]: Sorry about that, a brew upgrade munged my code and I had to bring it back from my repo, forgot to add that one back in.

Submitted.

**Comment by vitorgalvao** [comment]
Welcome, and thank you for the submission. Tip: use `x` in checklists (no spaces inside the brackets) and [Github will make them clickable](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments).

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**